security/docker/Dockerfile.node-agent-k8s

```
@@ -1,4 +1,5 @@
 FROM istionightly/base_debug
 ADD node_agent_k8s /
```

https://docs.docker.com/engine/articles/dockerfile_best-practices/#add-or-copy
For other items (files, directories) that do not require ADD’s tar auto-extraction capability, you should always use COPY.
```suggestion
COPY node_agent_k8s /
```

---

Done.